### ISSUE TITLE
Warnings when the package is installed 

### ISSUE THREAD

**Issue by ceyhunn** [h2qutc/angular-material-components#36]
When the package is installed, it shows following warnings:
```
warning " > @angular-material-components/datetime-picker@2.0.4" has incorrect peer dependency "@angular/platform-browser@0.0.0-NG".
warning " > @angular-material-components/datetime-picker@2.0.4" has incorrect peer dependency "@angular/common@0.0.0-NG".
warning " > @angular-material-components/datetime-picker@2.0.4" has incorrect peer dependency "@angular/core@0.0.0-NG".
warning " > @angular-material-components/datetime-picker@2.0.4" has incorrect peer dependency "@angular/forms@0.0.0-NG".
warning " > @angular-material-components/datetime-picker@2.0.4" has incorrect peer dependency "@angular/material@0.0.0-PLACEHOLDER".
warning " > @angular-material-components/datetime-picker@2.0.4" has incorrect peer dependency "@angular/cdk@0.0.0-PLACEHOLDER".
warning " > @angular-material-components/datetime-picker@2.0.4" has incorrect peer dependency "tslib@0.0.0-TSLIB".
```

**Comment by h2qutc** [comment]
Hi, can you provide un example of your package.json please ?

**Comment by ceyhunn** [comment]
Hi @h2qutc ,
```
"dependencies": {
        "@agm/core": "1.1.0",
        "@angular-material-components/datetime-picker": "^2.0.4",
        "@angular/animations": "9.1.1",
        "@angular/cdk": "9.2.0",
        "@angular/common": "9.1.1",
        "@angular/compiler": "9.1.1",
        "@angular/core": "9.1.1",
        "@angular/flex-layout": "9.0.0-beta.29",
        "@angular/forms": "9.1.1",
        "@angular/material": "9.2.0",
        "@angular/material-moment-adapter": "9.2.0",
        "@angular/platform-browser": "9.1.1",
        "@angular/platform-browser-dynamic": "9.1.1",
        "@angular/router": "9.1.1",
        "@angular/service-worker": "9.1.1",
        "@ngx-translate/core": "12.0.0",
        "@swimlane/ngx-datatable": "17.0.0",
        "@types/prismjs": "1.16.0",
        "angular-calendar": "0.28.2",
        "angular-in-memory-web-api": "0.9.0",
        "angular-resizable-element": "3.2.4",
        "prismjs": "1.19.0",
        "rxjs": "6.5.5",
        "tslib": "^1.10.0",
        "web-animations-js": "2.3.2",
        "zone.js": "0.10.2"
    },
    "devDependencies": {
        "@angular-devkit/build-angular": "0.901.1",
        "@angular/cli": "9.1.1",
        "@angular/compiler-cli": "9.1.1",
        "@angular/language-service": "9.1.1",
        "@angularclass/hmr": "2.1.3",
        "@types/jasmine": "3.5.2",
        "@types/jasminewd2": "2.0.8",
        "@types/lodash": "4.14.149",
        "@types/node": "^12.12.6",
        "codelyzer": "^5.2.1",
        "jasmine-core": "3.5.0",
        "jasmine-spec-reporter": "4.2.1",
        "karma": "4.3.0",
        "karma-chrome-launcher": "3.1.0",
        "karma-coverage-istanbul-reporter": "2.1.1",
        "karma-jasmine": "2.0.1",
        "karma-jasmine-html-reporter": "1.5.1",
        "protractor": "5.4.3",
        "ts-node": "8.3.0",
        "tslint": "5.18.0",
        "typescript": "3.8.3",
        "webpack-bundle-analyzer": "3.6.0"
    }
```

**Comment by ceyhunn** [comment]
@h2qutc , Did you reproduce this?

**Comment by carlabruti** [comment]
Hi, 

Same here. couldn't ng update with package already installed : #14 
Uninstall it and re installed it : 
```
npm WARN @angular-material-components/datetime-picker@2.0.4 requires a peer of @angular/platform-browser@0.0.0-NG but none is installed. You must install peer dependencies yourself.
npm WARN @angular-material-components/datetime-picker@2.0.4 requires a peer of @angular/common@0.0.0-NG but none is installed. You must install peer dependencies yourself.
npm WARN @angular-material-components/datetime-picker@2.0.4 requires a peer of @angular/core@0.0.0-NG but none is installed. You must install peer dependencies yourself.
npm WARN @angular-material-components/datetime-picker@2.0.4 requires a peer of @angular/forms@0.0.0-NG but none is installed. You must install peer dependencies yourself.
npm WARN @angular-material-components/datetime-picker@2.0.4 requires a peer of @angular/material@0.0.0-PLACEHOLDER but none is installed. You must install peer dependencies yourself.
npm WARN @angular-material-components/datetime-picker@2.0.4 requires a peer of @angular/cdk@0.0.0-PLACEHOLDER but none is installed. You must install peer dependencies yourself.
npm WARN @angular-material-components/datetime-picker@2.0.4 requires a peer of tslib@0.0.0-TSLIB but none is installed. You must install peer dependencies yourself.
```

Do i need to uninstall and re install it every time i need to update my dependencies ?

Thanks for your reply.

**Comment by ceyhunn** [comment]
Hi @carlabruti , you can use `--force` if you don't want to uninstall, re-install.

**Comment by joaodforce** [comment]
when I install the library I also get these warnings.

I do however have these dependencies im place, but with newer versions.
I'm using Angular 8

````➜  angular-fre-base-components git:(master) npm i
npm WARN ngx-mat-datetime-picker@2.0.3 requires a peer of @angular/platform-browser@~7.2.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-mat-datetime-picker@2.0.3 requires a peer of @angular/common@~7.2.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-mat-datetime-picker@2.0.3 requires a peer of @angular/core@~7.2.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-mat-datetime-picker@2.0.3 requires a peer of @angular/forms@~7.2.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-mat-datetime-picker@2.0.3 requires a peer of @angular/material@~7.3.7 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-mat-datetime-picker@2.0.3 requires a peer of @angular/cdk@~7.3.7 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-mat-datetime-picker@2.0.3 requires a peer of rxjs@~6.3.3 but none is installed. You must install peer dependencies yourself.

audited 22686 packages in 6.57s
found 786 vulnerabilities (767 low, 8 moderate, 11 high)
  run `npm audit fix` to fix them, or `npm audit` for details
➜  angular-fre-base-components git:(master)

**Comment by shalltalk** [comment]
See this https://github.com/h2qutc/angular-material-components/issues/53
Looks like for now this problem is fixed in 4.0.0 and up. Wish it could be fixed in previous versions so that it works angular 8&9.

**Comment by DanoTheNerd** [comment]
this is still causing warings and break the `ng update` comand, please add correct peer dependencies

*-Placeholder or *-NG is not a valid version

https://github.com/h2qutc/angular-material-components/blob/f147c1c9645167fb170bbca4aa9d91f757c2bb5a/projects/datetime-picker/package.json#L30-L36
